packages/cli/commands/project/dev.js

```diff
@@ -94,10 +107,42 @@ exports.handler = async options => {
       process.exit(EXIT_CODES.SUCCESS);
     }
   } else if (createNewSandbox) {
-    logger.log(
-      'Creating new sandboxes is not supported yet. Use "hs sandbox create" and then run this command again.'
-    );
-    process.exit(EXIT_CODES.SUCCESS);
+    try {
+      await validateSandboxUsageLimits(accountConfig, DEVELOPER_SANDBOX, env);
+    } catch (err) {
+      logErrorInstance(err);
+      process.exit(EXIT_CODES.ERROR);
+    }
```

Comment:
We do a limit validation for the default account and throw if there are no development sandboxes available to create